tests/test_jx/test_edge_1.py

```diff
@@ -436,15 +436,15 @@ def test_union_values(self):
                 "data": [
                     {"a": "x", "c": [2, 3]},
                     {"a": "y", "c": [4, 5, 6]},
-                    {"a": NULL, "c": [7, 8]}
+                    {"a": NULL, "c": [8, 7]}
```

Comment:
I believe you can use a Python `set` to indicate you do not care about order:

    {"a": NULL, "c": set([8, 7])}

or

    {"a": NULL, "c": {8, 7}}